components/brave_ads/core/internal/common/net/http/http_status_code_util.cc

```diff
@@ -40,6 +41,10 @@ std::optional<std::string> HttpStatusCodeClassToString(
 
 }  // namespace
 
+bool IsSuccessfulHttpStatusCode(const int http_status_code) {
+  return http_status_code >= net::HTTP_OK && http_status_code < 400;
```

Comment:
net::HTTP_BAD_REQUEST for 400?

Reply:
I did have this, but then thought having 400 made this code easier to read, because not everybody would know instantly that HTTP_BAD_REQUEST was 400. Your thoughts?

Reply:
PTAL as I changed to the above with comments. Thanks